src/test/java/org/jenkinsci/test/acceptance/recorder/TestRecorderRuleTest.java

```diff
@@ -28,11 +32,15 @@ public void shouldNotRecordSuccessTestExecutionByDefault() {
 
         //Clean the field
         outputFile.delete();
+
+        restoreRecorderOption(oldMode);
```

Comment:
:ant: agreed that would be the better style. Avoids duplicating this stuff in each test. Also would reset the option even if the test fails, unlike the current patch which neglects to use a `finally` block.